### PR TITLE
Add plans to runtime listParams

### PR DESCRIPTION
**Pull request by life0215**
**Description**

Changes proposed in this pull request:

- add Plans to runtime ListParameters

**Related issue(s)**

### CODE REVIEW

**Comment by life0215** [comment]
job pre-master-kcp-cli failed:
```
# github.com/kyma-project/control-plane/tools/cli/pkg/command
pkg/command/runtime.go:79:46: cmd.params.Plans undefined (type "github.com/kyma-project/control-plane/components/kyma-environment-broker/common/runtime".ListParameters has no field or method Plans)
Makefile:67: recipe for target 'build-windows' failed
```
Seems we need to create two separate PRs, one for server side change. another for client side change (after server side change is merged)

File: components/kyma-environment-broker/common/runtime/client_test.go [hunk 1]
```diff
@@ -59,6 +59,7 @@ func TestClient_ListRuntimes(t *testing.T) {
 			assert.ElementsMatch(t, params.RuntimeIDs, query[RuntimeIDParam])
 			assert.ElementsMatch(t, params.Regions, query[RegionParam])
 			assert.ElementsMatch(t, params.Shoots, query[ShootParam])
+			assert.ElementsMatch(t, params.Plans, query[PlanParam])
```
Comment (ebensom) [inline components/kyma-environment-broker/common/runtime/client_test.go:62]: You could add some plans above in `params` to test plan parameter passing of the client.

Comment (life0215) [reply components/kyma-environment-broker/common/runtime/client_test.go:63]: Added and tested, parsing works well

**Review by ebensom — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**